huaweicloud/services/acceptance/rms/resource_huaweicloud_rms_organizational_policy_assignment_test.go

```diff
@@ -0,0 +1,298 @@
+package rms
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/rms/v1/policyassignments"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getOrganizationalPolicyAssignmentResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	region := acceptance.HW_REGION_NAME
+
+	// getOrganizationalPolicyAssignment: Query the RMS organizational policy assignment
+	var (
+		getOrgPolicyAssignmentHttpUrl = "v1/resource-manager/organizations/{organization_id}/policy-assignments/{id}"
+		getOrgPolicyAssignmentProduct = "rms"
+	)
+	getOrgPolicyAssignmentClient, err := cfg.NewServiceClient(getOrgPolicyAssignmentProduct, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating Config client: %s", err)
+	}
+
+	getOrgPolicyAssignmentPath := getOrgPolicyAssignmentClient.Endpoint + getOrgPolicyAssignmentHttpUrl
+	getOrgPolicyAssignmentPath = strings.ReplaceAll(getOrgPolicyAssignmentPath, "{organization_id}",
+		state.Primary.Attributes["organization_id"])
+	getOrgPolicyAssignmentPath = strings.ReplaceAll(getOrgPolicyAssignmentPath, "{id}", state.Primary.ID)
+
+	getOrgPolicyAssignmentOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getOrgPolicyAssignmentResp, err := getOrgPolicyAssignmentClient.Request("GET", getOrgPolicyAssignmentPath,
+		&getOrgPolicyAssignmentOpt)
+
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RMS organizational policy assignment: %s", err)
+	}
+
+	return utils.FlattenResponse(getOrgPolicyAssignmentResp)
+}
+
+// Test the builtin policy assignment.
+func TestAccOrganizationalPolicyAssignment_basic(t *testing.T) {
+	var (
+		obj interface{}
+
+		rName = "huaweicloud_rms_organizational_policy_assignment.test"
+		name  = acceptance.RandomAccResourceNameWithDash()
+	)
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getOrganizationalPolicyAssignmentResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
```

Comment:
we should add a pre-check for Organizations service, please add acceptance.TestAccPreCheckOrganizationsOpen

Reply:
fixed